docs/guide/multitenant/service/storage.md

```diff
@@ -72,7 +72,7 @@ Azure Storage enables you to [provide encryption keys](/azure/storage/blobs/encr
 
 ### Monitoring
 
-When working with a multitenant solution, consider whether you need to [measure the consumption for each tenant](../considerations/measure-consumption.md), and define the specific metrics you need to track, such as the amount of storage used for each tenant (the capacity), or the number of operations performed for each tenant's data.
+When working with a multitenant solution, consider whether you need to [measure the consumption for each tenant](../considerations/measure-consumption.md), and define the specific metrics you need to track, such as the amount of storage used for each tenant (the capacity), or the number of operations performed for each tenant's data. You can also use [cost allocation](/azure/cost-management-billing/costs/allocate-costs) to track the cost of each tenant's usage and enable chargeback.  This can be useful if you have multiple subscriptions e.g., have a shared service in a different subscription than the tenant's subscription and need to manage and show cost accountability from all associated resources in multiple subscriptions.
```

Comment:
This is a great point, but I wonder if we should link to the [cost management and allocation article](https://learn.microsoft.com/azure/architecture/guide/multitenant/approaches/cost-management-allocation) instead of directly providing the guidance here?

Reply:
Also, if you do decide to keep this content here, we should remove "e.g." - MS Docs style is to not use Latin expressions. For "e.g." we usually say "for example," instead.

Reply:
Thanks @johndowns I agree, have updated to remove the additional description to reduce future maintenance.  I may create a PR on the [article](https://learn.microsoft.com/azure/architecture/guide/multitenant/approaches/cost-management-allocation) as it hasn't referred the cost allocation feature.